en/docs/reference/connectors/smpp-connector/smpp-connector-example.md

```diff
@@ -62,52 +63,16 @@ Follow the [Creating a REST API]({{base_path}}/develop/creating-artifacts/creati
-    
-        > **Note**: The properties should be added to the pallet before creating the operation.
-    
-    4. Add the property mediator to capture the `sourceAddress` value. The `sourceAddress` contains the source address of the SMS message. 
-   
-        - **Property Name**: `sourceAddress`
-        - **Property Value**: expression `json-eval($.sourceAddress)`
-   
-        <img src="{{base_path}}/assets/img/integrate/connectors/smpp-example-source-address-property.png" title="Add property mediators sourceAddress" width="800" alt="Add property mediators sourceAddress"/>
-    
-    5. Add the property mediator to capture the `message` values. The message contains the content of the SMS message.                  
-   
-        - **Property Name**: `message`
-        - **Property Value**: expression `json-eval($.message)`
-     
-        <img src="{{base_path}}/assets/img/integrate/connectors/smpp-example-message-property.png" title="Add values to capture message" width="800" alt="Add values to capture message"/>  
-      
-    6. Add the property mediator to capture the `destinationAddress` values. The message contains the content of the SMS message.                  
-       
-        - **Property Name**: `destinationAddress`
-        - **Property Value**: expression `json-eval($.destinationAddress)`
-         
-        <img src="{{base_path}}/assets/img/integrate/connectors/smpp-example-destination-address-property.png" title="Add values to capture destinationAddress" width="800" alt="Add values to capture destinationAddress"/>  
-    
-    7. Add the `sendSMS` operation in the `SMPP` connector. Select the previously created connection. Provide the following expressions for source address, destination address, and message as below. 
-        - **Source Address** - expression `$ctx:sourceAddress`
-        - **Destination Address** - expression `$ctx:destinationAddress` 
-        - **Message** - expression `$ctx:message`
+    4. Add the `sendSMS` operation in the `SMPP` connector. Select the previously created connection. Provide the following expressions for source address, destination address, and message as below. 
+        - **Source Address** - expression `payload.sourceAddress`
+        - **Destination Address** - expression `payload.destinationAddress` 
+        - **Message** - expression `payload.message`
 
-        <img src="{{base_path}}/assets/img/integrate/connectors/smpp-example-again-add-send-sms-operation.png" title="Again add send sms operation" width="800" alt="Again add send sms operation"/> 
-        <img src="{{base_path}}/assets/img/integrate/connectors/smpp-example-select-sms-conn.png" title="Select created SMSC connection" width="400" alt="Select created SMSC connection"/> 
+        To store the response of the operation in the message context, select "Overwrite Message Body" in the Output Section. This will allow you to send the response back to the user as a response of the API invocation.
```

Comment:
```suggestion
        To store the response of the operation in the message body, select **Overwrite Message Body** in the Output Section. This will allow you to send the response back to the user as a response of the API invocation.
```

Reply:
In the sample XML config this is set to false. https://github.com/wso2/docs-mi/pull/1607/files#diff-62d2e42e83e128a0f4c5a805d158fd9472ea69a8a75f2f070ed3f9b02d3b690bR90. Please check and update.